sublime_linter.py

```diff
@@ -138,7 +134,6 @@ def on_pre_close(self, view):
             self.loaded_views,
             self.linted_views,
             self.view_syntax,
-            persist.errors,
```

Comment:
We need to add some logic to remove `bid` from `errors` when it's not used anymore.

Reply:
This can be tricky. Generally we can multiple views into the same buffer. So we can remove only if this view here is the last one. But one has to check/test how we can iterate here over window.views() for example. Is this view here still in window.views(). What if the user closes the window?

So, that's probably tricky, then you should make an issue

Reply:
#900